client/src/assets/js/setAuthorizationToken.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
```

Comment:
You can have this function in a `util` file or directory along with other utility functions the app frequently uses